### PR TITLE
Create TABLE.md

### PR DESCRIPTION
**Pull request by pabloalmunia**
This is a simple table with descriptor parameter and descriptor return

### CODE REVIEW

**Review by littledan — COMMENTED** [review]
Could you post this as a markdown table rather than an HTML table?

**Review by littledan — COMMENTED** [review]
There's still plenty of markup here; sup tags are fine, but if you could avoid the rest (backtick for code, just not bothering with nbsp alignment) it'd be good.

**Comment by pabloalmunia** [comment]
I remove `&nbsp;` but now the first column hasn't indentation. 

I keep `<br/>` because this code is necessary for new line into markdown table.

File: TABLE.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+|`  descriptor:{`                  | -                                         | <sup>9</sup>                             | <sup>9</sup>                             | <sup>9</sup>                             |
+|`    value:`                      | -                                         |*method function*                         | -                                        | - <sup>10</sup>                          |
+|`    get:`                        | -                                         | -                                        | -                                        |*getter*                                  |
+|`    set:`                        | -                                         | -                                        | -                                        |*setter*                                  |
+|`    writable:`                   | -                                         |`true \|\| false`                         |`true \|\| false`                         | - <sup>10</sup>                          |
```
Comment (littledan) [inline TABLE.md]: There seems to be some kind of rendering issue with the way || is used.

File: TABLE.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+|`  kind:`                   |`"class"`                                 |`"method"`                                   |`"field"`                                            |`"method"` <sup>4</sup>                              |
+|`  elements:`               |*Array of member descriptors* <sup>1</sup>| -                                           | -                                                   | -                                                   |
+|`  key:`                    | -                                        |  *method name*                              |*method name*                                        |*property name*                                      |
+|`  placement:`              | -                                        |`"prototype" \|\| "static"`                  |`"own" \|\| "static"`                                |`"prototype" \|\| "static"`                          |
+|`  initializer:`            | -                                        | -                                           |*function than return the initial value* <sup>2</sup>| -                                                   |
```
Comment (littledan) [inline TABLE.md]: "function than return the initial value" doesn't make sense to me.

File: TABLE.md (hunk 1 above)
Comment (littledan) [reply TABLE.md]: Seems fixed

**Review by littledan — COMMENTED** [review]
Note that you can replace a field decorator with a method descriptor, or a getter, or vice-versa, but you can't interchange those with classes.

File: TABLE.md (hunk 2 above)
Comment (pabloalmunia) [reply TABLE.md]: Yes, it's a very strange expression, but my English is very pour. I will change this frase with "Function used to set the initial value of the field".

**Review by littledan — APPROVED** [review]

**Comment by littledan** [comment]
Thanks!

**Pull request merged**